command/crypto/change-pass.go

```diff
@@ -60,6 +71,16 @@ $ step crypto change-pass key.jwk --out new-key.jwk
 				Usage: "The <file> new encrypted key path. Default to overwriting the <key> positional argument",
 			},
 			flags.Force,
+			cli.BoolFlag{
+				Name:   "insecure",
+				Hidden: true,
+			},
```

Comment:
Although we're not using it consistently we should probably use `flags.Insecure` for this.